include/yarpWholeBodyInterface/yarpWholeBodyInterface.h

```diff
@@ -35,7 +35,7 @@ namespace wbi {
 
 namespace yarpWbi {
     class yarpWholeBodyActuators;
-    class yarpWholeBodyModel;
+    class yarpWholeBodyModelV1;
```

Comment:
If you are intersted, the V2 KDL-free version is available at https://github.com/traversaro/yarp-wholebodyinterface/compare/implementWholeBodyModelWithoutKDL .